tests/qos/test_qos_sai.py

```diff
@@ -1059,11 +1059,11 @@ def testQosSaiLossyQueue(
             testParams=testParams
         )
 
-    @pytest.mark.parametrize("LossyVoq", ["lossy_queue_voq_1"])
+    @pytest.mark.parametrize("LossyVoq", ["lossy_queue_voq_1", "lossy_queue_voq_2"])
     def testQosSaiLossyQueueVoq(
         self, LossyVoq, ptfhost, dutTestParams, dutConfig, dutQosConfig,
             ingressLossyProfile, duthost, localhost, get_src_dst_asic_and_duts,
-            skip_check_for_hbm_either_asic
+            skip_src_dst_different_asic
```

Comment:
@zhixzhu / @rraghav-cisco can you check if this correct. i am not sure .

Reply:
This is correct. Skip LossyQueueVoq case only if src and dst's asic_types are different. 
Since Q200 doesn't evicts lossy to HBM and Q100 evicts lossy to HBM, the qos params are different. And current test framework gets qos param based on ingress port, lossy testing is on egress asic.